src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1651,6 +1651,8 @@
   "com.minecolonies.coremod.entity.citizen.sleep": "All colonists are tucked into bed.",
   "com.minecolonies.coremod.entity.citizen.raining": "I don't like staying out in the rain, my clothes are gonna get wet.",
   "com.minecolonies.coremod.entity.citizen.raid": "Help! The colony is being raided. I am running for shelter.",
+  "com.minecolonies.coremod.entity.citizen.sleeping": "zZzz... I'm sleeping... Why did you wake me up?",
```

Comment:
What is triggerd when they are on the way to bed?

Reply:
Its triggered at the same time the icon in the gui is, for consistency.

Reply:
So maybe something more generic like "zZzz... I want to sleep... Why are you bothering me?"

Reply:
Yeah makes sense, I'll change that.

Reply:
Wait no I remember now, It gets triggered here because that is the best place I could find, but the validator only allows it while the citizen actually sleeps.
So it only shows up while the citizen actually sleeps, is that something I should change?

Reply:
Yeah, probably better

Reply:
which?
having it show when they sleep or when they want to sleep?

Reply:
when they want to sleep

Reply:
ok I'll change that then, I thought this was the intended way

Reply:
done.